pkg/convert/trivy/trivy.go

```diff
@@ -199,3 +194,31 @@ func makeCPE(v *gabs.Container) string {
 		pkgName,
 		pkgVersion)
 }
+
+func getPackageType(r *gabs.Container) string {
```

Comment:
Probably the only way to get this data to the API but this kind of language specific mapping will take maintenance.